plugin.video.iplayerwww/default.py

```diff
@@ -145,7 +145,10 @@ def get_params():
     Radio.ListMostPopular()
 
 elif mode == 117:
-    Radio.ListFavourites(logged_in)
+    Radio.ListListenList(logged_in)
+
+elif mode == 199:
+    Radio.ListFollwing(logged_in)
```

Comment:
Yeah, that's a (double) typo. Since it is consistent with the function name, it will not cause any problems.

Good find, though!